README.md

```diff
@@ -37,6 +37,8 @@ The Text Blocks Project appreciates any [contributions](https://github.com/gtt-p
 
 ## Version History
 
+- 2.0.0 Support Redmine 5.0
+- 1.2.0 GitHub Release
```

Comment:
@dkastl Maybe better to use `Publish on GitHub` related with your another plugin's PR ?
https://github.com/gtt-project/redmine_admissions/pull/1

Reply:
Well, I modified above on the latest commit.